lib/collective/collectors/pgbouncer.rb

```diff
@@ -0,0 +1,42 @@
+module Collective::Collectors
+  class PGBouncer < Collective::Collector
+    requires :pg
+
+    collect do
+      group 'pgbouncer' do |group|
+        instrument group, 'stats'
+        instrument group, 'pools'
+        instrument group, 'clients'
+      end
+    end
+
+  private
+
+    def instrument(group, thing)
+      group.group thing do |group|
+        instrument_tuples group, show(thing)
+      end
+    end
+
+    def show(thing)
+      conn.exec "show #{thing};"
+    end
+
+    def instrument_tuples(group, tuples)
+      tuples.each do |tuple|
+        source = tuple.key?('database') ? tuple['database'] : ''
+        tuple.each do |k,v|
+          group.instrument(k, v, source: source) if instrumentable?(v)
```

Comment:
I think instrument truncates lines if they are too long? Is it ok to do all of these in the same line? Are any of these variable length?